packages/core/breadcrumb.js

```diff
@@ -1,21 +1,21 @@
 const { isoDate } = require('./lib/es-utils')
 
-class BugsnagBreadcrumb {
-  constructor (name = '[anonymous]', metaData = {}, type = 'manual', timestamp = isoDate()) {
+class Breadcrumb {
+  constructor (message = '[empty]', metadata = {}, type = 'manual', timestamp = isoDate()) {
```

Comment:
I will investigate removing this – it could save us some bundle size and remove code paths if we just prevent a breadcrumb from ever being created with an empty message.